.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .idea
 .DS_STORE
+*.pyc
+main.db
```

Comment:
There is allready some comments in the code about setting up but i will write a separate readme for details later.